api/server/router/network/backend.go

```diff
@@ -11,7 +11,6 @@ import (
 // to provide network specific functionality.
 type Backend interface {
 	FindNetwork(idName string) (libnetwork.Network, error)
-	GetNetworkByName(idName string) (libnetwork.Network, error)
```

Comment:
`GetNetworkByName()` is not used in `api/server/router`, and this one could be misleading as multiple networks with the same name may exist.